### DISCUSSION TITLE
SUPABASE_DB_URL from Deno.env is not working and does not match Docker Console Output

### DISCUSSION THREAD

**Discussion by erichjrusch** [supabase/supabase#30043] [Questions]
I am using Supabase locally on a Mac.

My SUPABASEDB_URL when calling Deno.env.get("SUPABASE_DB_URL") is different that what is on the dashboard.

Terminal dashboard output when calling supabase start:
<img width="557" alt="image" src="https://github.com/user-attachments/assets/b72b73e9-25e3-4486-94fd-5b813dfe5f6a">

Docker Console Output for Edge Functions
<img width="900" alt="image" src="https://github.com/user-attachments/assets/63f36ddf-5349-4a92-bb20-4e9fcb976357">

The dashboard one works when I hardcode it into the code the one I obtain from Deno.env is not working.

I cannot override the SUPABASE env variables.  How do I fix this issue?

**Comment by anil1kuppa** [comment]
https://github.com/supabase/cli/issues/1972#issuecomment-2885523628
